R.gitattributes

```diff
@@ -12,3 +12,4 @@
 *.Rmd	  text
 *.R  	  text
 *.Rproj text
+*.Rmd linguist-language=R
```

Comment:
R markdown is markdown with support for R - the entire file shouldn't be marked as R.